pkg/generic/descriptor/http.go

```diff
@@ -67,6 +152,32 @@ func NewHTTPResponse() *HTTPResponse {
 	}
 }
 
+// SetStatusCode implements http.ResponseSetter of dynamicgo
+func (resp *HTTPResponse) SetStatusCode(code int) error {
+	resp.StatusCode = int32(code)
+	return nil
+}
+
+// SetHeader implements http.ResponseSetter of dynamicgo
+func (resp *HTTPResponse) SetHeader(key, val string) error {
+	resp.Header.Set(key, val)
+	return nil
+}
+
+// SetCookie implements http.ResponseSetter of dynamicgo
+func (resp *HTTPResponse) SetCookie(key, val string) error {
+	// TODO: send Cookie along with Header
```

Comment:
Why is this commented out? If it's useless, pls remove it and add a comment explaining this

Reply:
fixed